test/Index/index_module_refs.swift

```diff
@@ -0,0 +1,30 @@
+// RUN: %target-swift-ide-test -print-indexed-symbols -enable-source-import -source-filename %s -I %S/Store/Inputs | %FileCheck %s
+
+import ClangModuleB
+// CHECK: [[@LINE-1]]:8 | module/C | ClangModuleB | c:@M@ClangModuleB | Ref | rel: 0
+import ClangModuleC.Sub1
+// CHECK: [[@LINE-1]]:8 | module/C | ClangModuleC | c:@M@ClangModuleC | Ref | rel: 0
```

Comment:
Here's a nice improvement you can make for how test checking works in this file:

* Match the USR and add it as a [FileCheck variable](
https://llvm.org/docs/CommandGuide/FileCheck.html#filecheck-variables). So instead of having `c:@M@ClangModuleC` make it `[[ClangModuleC_USR:c:@M@ClangModuleC]]`
* Later in the file, instead of repeating the same USR to match (`c:@M@ClangModuleC`) you match with the variable you defined earlier (`[[ClangModuleC_USR]]`).
* Do the same for the other USRs that get repeated, like for `SwiftModuleC`.

The benefit of this is that it reduces the duplication of the USR string. Later on, if for some reason we decide to change how the USRs for modules are formed, there will be much fewer places in the test cases that would be necessary to correct.

Reply:
Amazing feature. Thanks for the tip!